libcontainer/cgroups/fs2/fs2.go

```diff
@@ -129,6 +129,9 @@ func (m *manager) GetStats() (*cgroups.Stats, error) {
 }
 
 func (m *manager) Freeze(state configs.FreezerState) error {
+	if m.config.Resources == nil {
```

Comment:
See my comment on one of the earlier commits (should we guard against `m.config` being able to be `nil`?)

Reply:
I have removed checks for `m.config` being nil (see the last commit), as this is checked in `cgroups/manager.New`, and all the other code assumes that `m.config` (called `m.cgroups` in all the other managers except fs2) is non-nil.